test/LockableToken-creation_test.js

```diff
@@ -12,6 +13,35 @@ contract('LockableToken', ([owner, receiver, spender]) => {
   const lockTimestamp = Number(new Date()) / 1000
   const approveAmount = 10
   const nullAddress = 0x0000000000000000000000000000000000000000
+  const increaseTime = function(duration) {
```

Comment:
do you need both this and the `timeTravel` function?

Reply:
THanks @dave. Haven't used timeTravel. Will remove the include.